src/Core/Util/String/StringModifierInterface.php

```diff
@@ -28,6 +28,8 @@
 
 /**
  * Defines reusable methods for strings modifications.
+ *
+ * @method str2url(string $string, bool $allowAccentedChars)
```

Comment:
@PrestaShop/prestashop-core-developers Should I break the interface here or should I leave it that way ?